prompt.py

```diff
@@ -970,6 +970,9 @@ def main():
     parser.add_argument("-v", "--verbose", action="store_true", default=False,
                         help="Show smart-contract events by default")
 
+    # Where to store stuff
+    parser.add_argument("--datadir", action="store", help="Absolute path to prepend to paths for database directories")
```

Comment:
should we call it `--db-path` to make it more explicit? `--datadir` works too though.

Would change help to "Absolute path to use for database directories"